Cargo.lock

```diff
@@ -3626,3 +3866,52 @@ name = "zeroize"
 version = "1.7.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "525b4ec142c6b68a2d10f01f7bbf6755599ca3f81ea53b8431b7dd348f5fdb2d"
+
+[[package]]
+name = "zip"
```

Comment:
Uff, so many new dependencies. Would you mind having a look at https://docs.rs/crate/zip/0.6.6/features and removing all features that we don't need? We probably only need a very specific subset